### PR TITLE
[screensaver.bing] 1.0.0

### PR DESCRIPTION
**Pull request by Lunatixz**
Explore breathtaking images from the U.S. Bing homepage over the past 5
years. ...

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: screensaver.bing/addon.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        <summary lang="en_GB">Explore breathtaking images from the U.S. Bing homepage over the past 5 years. ... </summary>
+        <description lang="en_GB"></description>
+        <license>GNU GENERAL PUBLIC LICENSE. Version 3, June 2007</license>
+        <platform>all</platform>
+        <source>https://github.com/Lunatixz/XBMC_Addons/tree/master/screensaver.bing</source>
+        <news></news>
+        <assets>
+            <icon>icon.png</icon>
+            <fanart>fanart.jpg</fanart>
+            <screenshot>screenshot01.png</screenshot>
```
Comment (enen92) [inline screensaver.bing/addon.xml]: Those can probably be moved to the resources folder

File: screensaver.bing/default.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+#
+# You should have received a copy of the GNU General Public License
+# along with Bing ScreenSaver.  If not, see <http://www.gnu.org/licenses/>.
+
+import gui
+import xbmcaddon
+
+# Plugin Info
+ADDON_ID       = 'screensaver.bing'
+REAL_SETTINGS  = xbmcaddon.Addon(id=ADDON_ID)
```
Comment (enen92) [inline screensaver.bing/default.py]: the id is not needed

File: screensaver.bing/gui.py [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+# You should have received a copy of the GNU General Public License
+# along with Bing ScreenSaver.  If not, see <http://www.gnu.org/licenses/>.
+
+import urllib, urllib2, socket, json, os, random
+import xbmc, xbmcaddon, xbmcvfs, xbmcgui
+from simplecache import use_cache, SimpleCache
+
+# Plugin Info
+ADDON_ID       = 'screensaver.bing'
+REAL_SETTINGS  = xbmcaddon.Addon(id=ADDON_ID)
```
Comment (enen92) [inline screensaver.bing/gui.py]: same here

File: screensaver.bing/resources/skins/default/skin.xml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<skin>
```
Comment (enen92) [inline screensaver.bing/resources/skins/default/skin.xml]: It's the first time I have seen this. Is this file actually used?

Comment (Lunatixz) [inline screensaver.bing/resources/skins/default/skin.xml]: Not really sure one of the legacy things... i've always left alone. I'll remove it.

File: screensaver.bing/gui.py (hunk 3 above)
Comment (Lunatixz) [inline screensaver.bing/gui.py]: if it's okay i'd like to leave this as is, THX

**Pull request merged**